.gitignore

```diff
@@ -1,10 +1,11 @@
-target/
-pom.xml.tag
-pom.xml.releaseBackup
-pom.xml.versionsBackup
-pom.xml.next
-release.properties
-dependency-reduced-pom.xml
-buildNumber.properties
-.idea/
-*.iml
+/target/
+/project/*/
+#pom.xml.tag
+#pom.xml.releaseBackup
+#pom.xml.versionsBackup
+#pom.xml.next
+#release.properties
+#dependency-reduced-pom.xml
+#buildNumber.properties
+/.idea/
+/*.iml
```

Comment:
I think some of the ignores are from Java project legacy.
On the sbt specifics, I tend to do as above (please ask for more if you like).
In general, I've found anchoring rules to / when they really apply to the root only (like /.idea/) is good. Explicit.